Sources/Apodini/Relationships/RelationshipInheritance.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Created by Andreas Bauer on 18.01.21.
+//
+
+/// A `RelationshipInheritance` can be used to indicate that the annotated `Content` type
+/// inherits Relationships from the specified target type.
+public struct RelationshipInheritance<From, To>: RelationshipDefinition {
```

Comment:
nit: Shouldn't `To` here also be constrained to `ResponseTransformable`?

Reply:
Same as in https://github.com/Apodini/Apodini/pull/147#discussion_r568601289